web/elm/src/Dashboard/Grid/Constants.elm

```diff
@@ -21,17 +21,21 @@ cardBodyHeight =
 
 padding : number
 padding =
-    25
+    32
 
 
 cardHeaderHeight : number -> number
 cardHeaderHeight numRows =
-    20 + 30 * numRows
+    32 + 21 * numRows
```

Comment:
I don't think we want 21 here - we should give some padding between the rows when we have multiple header rows.

On `master`:
<img width="291" alt="Screen Shot 2021-02-17 at 12 35 42 PM" src="https://user-images.githubusercontent.com/26583442/108243962-a79c6680-711c-11eb-98e7-5a95ef5a096d.png">


With this change:
<img width="296" alt="Screen Shot 2021-02-17 at 12 35 15 PM" src="https://user-images.githubusercontent.com/26583442/108243902-981d1d80-711c-11eb-9c49-3aec27b3305e.png">

Reply:
@matthewpereira from the Figma render we see the height for one row header is 21px. When there are multiple rows what vertical spacing should be in between?

Reply:
![Screen Shot 2021-02-19 at 9 40 56 AM](https://user-images.githubusercontent.com/1585949/108518694-d63e4c80-7296-11eb-9261-622c6630145e.png)
![Screen Shot 2021-02-19 at 9 41 05 AM](https://user-images.githubusercontent.com/1585949/108518695-d6d6e300-7296-11eb-9718-02b292870579.png)

these are samples when vertical spacing is set to 8px between header rows